### DISCUSSION TITLE
FastAPI for Julia

### DISCUSSION THREAD

**Discussion by rozeappletree** [fastapi/fastapi#7290] [Questions] [answered]
Hi,

 I felt elated while working with FastAPI. But, unfortunately my backend is entirely written in Julia. 

1. Can I somehow expose it through Python FastAPI?
2. Is there Julia wrapper for FastAPI ? If not, are there any plans to extend FastAPI to Julia?

Thanks,
Rakesh

**Comment by raphaelauv** [comment, marked as answer]
1) Use a python binding lib doing it for you [[julia_doc]](https://docs.julialang.org/en/v1/manual/embedding/)
2) Never heard about something like this, calling julia code from python fastAPI code is far less complicated.

**Comment by rozeappletree** [comment]
I see, I closing the issue. Thanks!

**Comment by tiangolo** [comment]
Thanks for the help here @raphaelauv ! :clap: :bow: 

Thanks for reporting back and closing the issue @rakesh4real  :+1:
